scripts/golangci.yml.in

```diff
@@ -164,3 +164,6 @@ linters:
     - godox
     - wsl
     - funlen
+    - testpackage
+    - goerr113
```

Comment:
This can be removed isn't it? There was a pr to address error comparison issue

Reply:
This is added to disable the `goerr113` linter which otherwise may throw multiple warnings. Removing this will leave us no choice but to address all goerr warnings.

Reply:
can you please open an issue to address these errors? I think then we can enable it

Reply:
Sure, I'll open an issue for the same.

Reply:
Also mark it here with a `TODO: enable goerr113, see ceph/ceph-csi#1234`